modules/Vtiger/models/Install.php

```diff
@@ -213,26 +225,44 @@ public function install()
         }
     }
 
+    public function isInstalledModule(): bool
```

Comment:
1. PHPDoc missing
2. This method does not only check whether the module is installed or not, but it also sets the module as installed, although it isn't (yet). It should only check; module should be set as installed at the end of installation process. During the installation could unexpected errors happen, and this method will wrongly set the module as installed before it actually happens.